e2e/tests/simple-v4-ingress.bats

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bats
+
+# Note:
+# These test cases, simple, will create simple (one policy for ingress) and test the 
+# traffic policying by ncat (nc) command. In addition, these cases also verifies that
+# simple iptables generation check by iptables-save and pod-iptable in multi-networkpolicy pod.
+
+
+setup() {
```

Comment:
What about using `setup_file()` instead of relying on test-order? 
https://bats-core.readthedocs.io/en/stable/writing-tests.html#setup-and-teardown-pre-and-post-test-hooks

Doing that allows to run a single test using `bats --filter "<test name>"`.
It also cleans the way setup actually works, as the `setup()` instance of `@test "setup simple test environments"` does nothing.

Reply:
make sense. Changed.

Reply:
hmm. changing setup_file() may be not good to us to troubleshooting because setup_file() or setup() does not verify return code. At the start of e2e tests, we should create environment and wait for the resource ready. Write it in a test case could be more simple even if we may rely on test-order.

Reply:
ok! not a big deal anyway.